python/ray/train/v2/api/data_parallel_trainer.py

```diff
@@ -74,14 +83,45 @@ def __init__(
         if metadata is not None:
             raise DeprecationWarning(_GET_METADATA_DEPRECATION_MESSAGE)
 
+        tag_train_v2_trainer(self)
```

Comment:
Do you think this should be in `fit` instead?

Reply:
No strong opinion